### PR TITLE
[1LP][RFR] Updating Server Settings to get WebMKS SDK downloaded and extracted on Appliance

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Extending__ Server settings to accommodate changes in the WebMKS Console. Now in order to find WebMKS Console option on the server settings page, we need to download WebMKS SDK and extract it under /var/www/miq/vmdb/public/ in a new webmks directory. That enables the WebMKS Option in the UI. 

These features are currently in 5.9 only. Soon to be added in CFME 5.8 as well.

{{ pytest: cfme/tests/infrastructure/test_webmks_console.py --use-provider vsphere65-nested }}

### CODE REVIEW

File: cfme/configure/configuration/server_settings.py [hunk 1]
```diff
@@ -328,6 +328,13 @@ def update_vmware_console(self, updates, reset=False):
                         product_name=self.appliance.product_name,
                         version=self.appliance.version
                     )
+            if value == 'VMware WebMKS':
```
Comment (rlbabyuk) [inline cfme/configure/configuration/server_settings.py]: **Required:** Please move this under  if name == 'console_type', to check console type only if this is console

**Comment by quarckster** [comment]
PRT failed

**Comment by kedark3** [comment]
@quarckster All the PRT failures are at much later state, and has nothing to do with changes in the PR. I have started re-test on the PRT.

**Comment by kedark3** [comment]
Upstream failure is one of the sporadic failures, I do not think there is any action required.

File: cfme/configure/configuration/server_settings.py [hunk 2]
```diff
@@ -328,6 +328,14 @@ def update_vmware_console(self, updates, reset=False):
                         product_name=self.appliance.product_name,
                         version=self.appliance.version
                     )
+                if value == 'VMware WebMKS' and self.appliance.version > '5.9':
```
Comment (quarckster) [inline cfme/configure/configuration/server_settings.py]: **REQUIRED**
```python
if value == 'VMware WebMKS' and self.appliance.version >= '5.9':
```

**Comment by kedark3** [comment]
All the PRT failures are at much later state, and has nothing to do with changes in the PR.

**Comment by quarckster** [comment]
@kedark3  I wasn't able to successfully run `cfme/tests/infrastructure/test_webmks_console.py` from your branch. I got 
```text
cfme/tests/infrastructure/test_webmks_console.py::test_webmks_vm_console[vsphere65-nested] ERROR

    @pytest.fixture(scope="module")
    def configure_vmware_console_for_test(appliance):
        """Configure VMware Console to use VNC which is what is required for the HTML5 console."""
>       appliance.server.settings.update_vmware_console({'console_type': 'VMware WebMKS'})

cfme/tests/infrastructure/test_webmks_console.py:45:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
cfme/configure/configuration/server_settings.py:333: in update_vmware_console
    .format(conf.cfme_data.vm_console.webmks_console.webmks_sdk_download_url))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = AttrDict([('cert', AttrDict([('country', 'US'), ('state', 'North Carolina'), (... ('organizational_unit', 'QE'), ('install_dir', '/var/www/miq/vmdb/certs')]))])
k = 'webmks_console'

    def __getattr__(self, k):
>           if not (k.startswith('__') or k.startswith('_OrderedDict__')): return self[k]
E           KeyError: 'webmks_console'

../lib/python2.7/site-packages/lya/lya.py:68: KeyError
```

**Comment by kedark3** [comment]
@quarckster  please update your YAMLs in order to run this successfully. There was new key added to cfme_data.yaml

**Comment by kedark3** [comment]
@jkrocil Yes. So when you unzip the file, no server restart is required. If you navigate to Configuration page after unzipping the file, the "WebMKS" option becomes available in the dropdown. And if you are already on the Configuration page, just refresh it after you unzip. That's all. Thanks for Merge.

**Pull request merged**